---
title: "Analysis Report 2: Your Title Here"
author: "Isabella Finney"
date: "November 20, 2017"
output: github_document
bibliography: references.bib
csl: bioinformatics.csl
---

# Introduction

The history of smoking in the world dates back to at least 4,000 BC. For much of history, the world used tobacco for different purposes such as trade, medicinal alleviation, and hallucinogenic uses. In the 20th century, smoking became a part of the mainstream culture as it was widely advertised and glamorized in the media by huge tobacco industries. It was not until the late 1980s, that research on the effects of tobacco and smoking started and soon after many results showed the gruesome results that tobacco and heavy smoking causes on the body as well as the ecosystem. From then on, policies, organizations, and campaigns have been enacted all over the world in an effort to reduce the negative effects it has caused to both people and the environment [@koh2007making].  
The effects of smoking are damaging to nearly every organ in the body. Years of extensive research shows that exposure to smoking leads to years of damage to the body. Unfortunately, it is also the leading preventable cause of cancer and death in the world [@centers2010tobacco].  The development and application of modern molecular methods has made it possible for scientists to more fully understand how exactly smoking increases the risk for contracting various diseases and cancers in the body [@powell2013association]. After doing much research, scientists were able to identity differences in molecular characteristics between smokers and nonsmokers who were diagnosed with lung cancer. For example, Massion et al. located structural differences in the genomes of smokers and nonsmokers with adenocarcinomas which help propose the idea that smoking affects the molecular pathways that form during cancer development [@massion2008smoking].
	One of the most useful tools for investigating lung cancer development is RNA-seq technology. This technology helps in providing a tool for transcriptome analysis in studies. It presents a great utility for detecting tumor-associated genes as well as identifying of cancer pathogenesis because of its sensitivity and has less background noise from image analysis [@khoury2013landscape]. The paper done by Li et al. examined the pair-wise comparison analysis on samples collected from a previous study that did RNA-seq analysis on paired normal and tumor tissues. This study aimed to distinguish the differences between gene expression profile between smoking and nonsmoking patients in the hopes of creating better individualized treatments [@li2015rna]. Using the dataset from this study, I investigated whether gender plays a risk factor in being more susceptible to contracting lung cancer. With the use of this dataset, I hypothesized that men would exhibit higher levels of gene expression in different cancer stages. I made this hypothesis due to the history associated with it being more acceptable for men to smoke. I also factored in statistics showing that overall, men had a higher mortality rate than women in cigarette smoking [@hammond1966smoking].
	

# Methods

## Sample origin and sequencing

Li et al. downloaded transcriptome sequencing data samples from the Gene Expression Omnibus or GEO [@li2015rna]. The data samples were taken from a study done by Seo et al. that took 68 patients with lung adenocarcinoma (34 non-smokers and 34 smokers). They restricted age to ensure comparability in age groups [@seo2012transcriptional]. To compare the genes identified within the nonsmokers, Li et al. also imported an independent RNA-seq data set from GEO. Gene expression analysis was conducted on 136 RNA-seq bam files from each patient and pair-wise concordance were computed to confirm the paired nature of the tumor and normal tissues. The RNA-seq reads were aligned to human genome assembly via Tophat. HTSeq counted the reads and R Bioconductor edgeR preformed differential analysis with a linear model. The model aided in identifying the different genes expressed in tumor and normal tissues from smokers and nonsmokers. A logFC value was placed to count the significant genes. Pathway analysis was then applied to identify the relationships of the genes with varied expression.

## Computational

The process to computationally parse and analyze the data gathered was accomplished using a systematic workflow from different scripts.The first step was to download an SRA toolkit for the RNA-seq analysis from NCBI. This allowed it to configure and download the directory and use aspera to quickly download the fastq files. Next, a script was created to download a reference genome with biomartr package from Bioconductor. This package can also annotate the wide range of genes from NCBI and achieve computational reproducibility [@durinck2005biomart]. A for loop script was used to convert the paired-end SRA files to fastq files via fasta-dump. Another script was created that incorporated an aligner called sailfish to build and index for file for a reference genome. Sailfish quantifies the abundance of annotated RNA isoforms and provides faster quantification because it avoids mapping reads [@patro2014sailfish]. Fastqc was run in order to preform quality checks on all of the fastq files. Next, the Trimmomatic  tool to clip off the Illumina adapters and any low quality reads below 20 in a window size of 4. Trimmomatic is such a valuable tool because it can handle preprocessing on paired-end data, with or without a reference [@bolger2014trimmomatic]. If any sequences were untrimmed in the first round, another script was made to retrim those sequences. Sailfish was run to count the reads from trimmed reads from Trimmomatic quality checks using 88 cores. The next R script made a conversion from transcripts to a gene ID mapping table. The result is a table with gene names of rows and columns of samples as well as a number of normalized counts of each gene in each sample. The last script was meant to consolidate all of the counts together and merge it with the metadata so that output comes in a nicely compressed csv format for easy readability and reproducibility for investigation in the Results section.  

# Results

```{r load-libraries, message = FALSE, echo = FALSE}
# Be sure to install these packages before running this script
# They can be installed either with the intall.packages() function
# or with the 'Packages' pane in RStudio

# load general-use packages
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")
library("magrittr")

# this package allows for the easy inclusion of literature citations in our Rmd
# more info here: https://github.com/crsh/citr
# and here:
# http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
library("citr")
```

```{r load-data, message = FALSE, echo = FALSE}
# load the dataset from a compressed binary file
# it gets loaded as an object called "final_table"
# this has 3.4 million rows...so you will need to be thoughtful about
# how you analyze the data so that you don't overwhelm your laptop
load("output/final_compiled_counts/joined_count_data.RData")

# changes all columns names to lower case so Travis doesn't complain
names(final_table) %<>% tolower

# test that it loaded correctly before proceeding
stopifnot(exists("final_table"))
```

```{r make-summary-table, echo = FALSE}
# There are many many genes in this dataset, so we can
# subset it down to just a few here to look for interesting patterns
# in the most highly expressed
top_15 <- final_table %>%
  group_by(genename) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count)) %>%
  head(n = 15)

# then we can use the `kable()` function to make a nicely formatted
# markdown table
top_15 %>%
  kable()
```

**Table 1**: The most highly expressed genes included are *SFTPB* and *EEF1A1*.

```{r boxplot-of-highly-expressed-genes-by-smoking-status, echo = FALSE}
top_genes <- top_15 %>%
  ungroup() %>%
  select(genename) %>%
  unique() %>%
  pull()

final_table %>%
  filter(genename %in% top_genes) %>%
  ggplot(aes(x = genename,
             y = counts_lengthscaledtpm,
             fill = gender)) +
    geom_boxplot() +
    facet_wrap(~smoking_status) +
    xlab("Gene Name") +
    ylab("Scaled Read Counts per Gene") +
    ggtitle("Read Counts per Gene by Gender and Smoking Status") +
    theme_bw() + # simplifies theme
    theme(axis.text.x = # rotates x axis labels vertically
            element_text(angle = 90,
                         hjust = 1))
```

**Figure 1**: In this study, smoking status shows that overall, more males admitted to being current smokers or having smoked previously.


```{r make-barplot-of-highly-expressed-genes, echo = FALSE}
# this code uses the same data as above, but use it to make a
# barplot - remember geom_col() is just like
# when you use geom_bar(stat = "identity")
final_table %>%
  filter(genename %in% top_genes) %>%
  group_by(genename, gender) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  ggplot(aes(x = genename,
             y = mean_count,
             fill = gender)) +
    geom_col(position = "dodge")
```

**Figure 2**: From the 15 most highly expressed genes, males, on average, show a higher expression count than females do. 

```{r bar-plot-of-highest-expressed-genes-by-cancer-stage, echo = FALSE}
top_genes <- top_15 %>%
  ungroup() %>%
  select(genename) %>%
  unique() %>%
  pull()

final_table %>%
  filter(genename %in% top_genes) %>%
  ggplot(aes(x = genename,
             y = counts_lengthscaledtpm,
             fill = cancer_stage)) +
    geom_col(position = "dodge")
```

**Figure 3**: This figure compares the highest expression levels of these 15 genes at each cancer stage. The majority of the genes show highest expression levels at stage 2B cancer. 

```{r make-boxplot-of-highly-expressed-genes-by-gender, echo = FALSE}
# here we just want to pull out the unique gene names and turn
# them into a vector so we can use it below to make a boxplot
# we use the pull() funtion to get this as a vector, just like
# we did when making histograms several weeks ago
top_genes <- top_15 %>%
  ungroup() %>%
  select(genename) %>%
  unique() %>%
  pull()

# now we need to filter from the full data set again, because
# we don't just want summary data, we want all the data in
# order to make boxplots
final_table %>%
  filter(genename %in% top_genes) %>%
  ggplot(aes(x = genename,
             y = counts_lengthscaledtpm,
             fill = gender)) +
    geom_boxplot() +
    facet_wrap(~cancer_stage, ncol = 2) +
    xlab("Gene Name") +
    ylab("Scaled Read Counts per Gene") +
    ggtitle("Read Counts per Gene by Gender and Cancer Stage") +
    theme_bw() + # simplifies theme
    theme(axis.text.x = # rotates x axis labels vertically
            element_text(angle = 90,
                         hjust = 1)) +
  scale_y_log10()
```

**Figure 4**: Comparison of high expression levels by gender and cancer stage. Over every stage of cancer, expression levels in each gene are often slightly higher in males. Interestingly at stage 2B cancer, high expression levels are only seen in males. Male expression general grows higher at later cancer stages.

  
# Discussion

Add around 1-2 pages interpreting your results and considering future directions one might take in analyzing these data.

# Sources Cited
